core/templates/dev/head/pages/exploration_editor/editor_tab/test_interaction_modal_directive.html

```diff
@@ -1,14 +1,14 @@
+<style>
+.submit-answer-btn {
```

Comment:
Indent by 2.

/cc @apb7 @kevinlee12

Reply:
The indent rule has been already added in Issue #5097. I will try to make a fix for this during the weekend since it seems to be occurring at a number of places at the moment.

Reply:
Done.